test/onnx/onnx_import.in.cpp

```diff
@@ -1348,11 +1377,29 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, model_softplus_infinity)
                                      std::numeric_limits<float>::infinity(),
                                      std::numeric_limits<float>::infinity()}};
 
-    Outputs outputs{execute(function, inputs, "${BACKEND_NAME}")};
-    for (float v : outputs.front())
-    {
-        EXPECT_TRUE(std::isinf(v));
-    }
+    std::vector<float> expected_output{std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity(),
+                                       std::numeric_limits<float>::infinity()};
```

Comment:
I'm not sure whether this is the right way to check for infinity. @rkimballn1 Do you think this is OK?

Reply:
I'm not sure too, this is what I actually wanted to dicsuss on the review.
Maybe this test doesn't have to check so many infinities.
As we can see on the diff it was previously check in loop with `std::isinf()`
  ```
for (float v : outputs.front())
{
        EXPECT_TRUE(std::isinf(v));
}
```

Reply:
It looks like there are 13 elements in both input and output because the model used for this test was originally designed for another UT -  the `model_softplus` test which is one above.

To make this one cleaner you can replace the creation of both inputs and expected outputs with this: `std::vector<float>(13, std::numeric_limits<float>::infinity());`.